src/main/java/seedu/storage/IngredientList.java

```diff
@@ -0,0 +1,12 @@
+package seedu.storage;
+
+import seedu.data.ingredient.Ingredient;
+import java.util.ArrayList;
+
+public class IngredientList {
+    protected ArrayList<Ingredient> ingredientList;
+
+    public IngredientList(ArrayList<Ingredient> ingredientList) {
+        ingredientList = new ArrayList<>(100);
```

Comment:
I think we don't have to specify the (100) but I think can just merge first and it should be:

ingredientsList = new ArrayList<Ingredient>();

Reply:
okay we can update that later